driver/src/examples/documentation/ChangeStreamSamples.java

```diff
@@ -0,0 +1,173 @@
+        MongoDatabase database = mongoClient.getDatabase("testChangeStreams");
+        database.drop();
+        sleep();
+
+        // Select the collection to query.
+        MongoCollection<Document> collection = database.getCollection("documents");
+
+        /*
+         * Example 1
+         * Create a simple change stream against an existing collection.
+         */
+        System.out.println("1. Initial document from the Change Stream:");
+
+
+        // Create the change stream cursor.
+        MongoCursor<Document> cursor = collection.watch().iterator();
+
+        // Insert a test document into the collection.
+        collection.insertOne(Document.parse("{username: 'alice123', name: 'Alice'}"));
+        Document next = cursor.next();
+        System.out.println(next.toJson());
+        cursor.close();
+        sleep();
+
+        /*
+         * Example 2
+         * Create a change stream with 'lookup' option enabled.
+         * The test document will be returned with a full version of the updated document.
+         */
+        System.out.println("2. Document from the Change Stream, with lookup enabled:");
+
+        // Create the change stream cursor.
+        cursor = collection.watch().fullDocument(FullDocument.UPDATE_LOOKUP).iterator();
+
+        // Update the test document.
+        collection.updateOne(Document.parse("{username: 'alice123'}"), Document.parse("{$set : { email: 'alice@example.com'}}"));
+
+        // Block until the next result is returned
+        next = cursor.next();
+        System.out.println(next.toJson());
+        cursor.close();
+        sleep();
+
+        /*
+         * Example 3
+         * Create a change stream with 'lookup' option using a $match and ($redact or $project) stage.
+         */
+        System.out.println("3. Document from the Change Stream, with lookup enabled, matching `update` operations only: ");
+
+        // Insert some dummy data.
+        collection.insertMany(asList(Document.parse("{updateMe: 1}"), Document.parse("{replaceMe: 1}")));
+
+        // Create $match pipeline stage.
+        List<Bson> pipeline = singletonList(
+                Aggregates.match(
+                        Filters.or(
+                                Document.parse("{'fullDocument.username': 'alice123'}"),
+                                Filters.in("operationType", asList("update", "replace", "delete"))
+                        )
+                )
+        );
+
+        // Create the change stream cursor with $match.
+        cursor = collection.watch(pipeline).fullDocument(FullDocument.UPDATE_LOOKUP).iterator();
+
+        // Forward to the end of the change stream
+        next = cursor.tryNext();
+
+        // Update the test document.
+        collection.updateOne(Filters.eq("updateMe", 1), Updates.set("updated", true));
+        next = cursor.next();
+        System.out.println(format("Update operationType: %s %n %s", next.get("updateDescription", Document.class).toJson(), next.toJson()));
```

Comment:
This isn't printing the 'operationType'. Maybe:
`System.out.println(format("Update change: %s", next.toJson()));`

Same for Replace and Delete below